app/controllers/organization_users_controller.rb

```diff
@@ -35,4 +35,10 @@ def change_role
     authorize org_user
     org_user.update(role: params[:role])
   end
+
+  def archive
+    org_user = OrganizationUser.find params[:organization_user_id]
+    authorize org_user
```

Comment:
This is common to all controller methods. Can you move this `before_action` filter?

Reply:
Done.